Makefile

```diff
@@ -78,10 +78,13 @@ ng_tests: ## Run Angular tests
 
 linters : ## Run linters
 	docker pull hub.adsw.io/library/pr-builder:3-x64
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh shellcheck pylint pep8
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests black
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests/functional flake8_pytest_style
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests/ui_tests flake8_pytest_style
+	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 \
+        /bin/bash -xeo pipefail -c "/linters.sh shellcheck pylint pep8 && \
+        /linters.sh -b ./tests -f ui_tests pylint && \
+        /linters.sh -b ./tests -f functional pylint && \
```

Comment:
I guess we should lint all tests folder here

Reply:
unfortunately it doesn't work yet
```
************* Module ui_tests.test_cluster_list_page

ui_tests/test_cluster_list_page.py:46: [E0401(import-error), ] Unable to import 'ui_tests.utils'
```